synfig-studio/src/gui/app.cpp

```diff
@@ -306,7 +306,7 @@ static int max_recent_files_=25;
 int    studio::App::get_max_recent_files()      { return max_recent_files_; }
 void   studio::App::set_max_recent_files(int x) {        max_recent_files_ = x; }
 
-static synfig::String app_base_path_;
+static synfig::String app_base_path_ ;
```

Comment:
I know it's a draft, but it's better to fix it sooner than later: unnecessary extra space.

Reply:
Np ! Will Take care of it :)